lib/podio/models/file_attachment.rb

```diff
@@ -1,8 +1,10 @@
 # @see https://developers.podio.com/doc/files
 class Podio::FileAttachment < ActivePodio::Base
   property :file_id, :integer
+  property :uuid, :string
   property :external_file_id, :string
   property :link, :string
+  property :uuid_link, :string
```

Comment:
Where is `uuid_link` being used?

Reply:
Its working without this change hence deleting this PR